pkgs/games/super-tux-kart/CMakeLists-system-libs.patch

```diff
@@ -0,0 +1,32 @@
+diff -ur a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt	2020-06-24 18:36:18.000829507 +0200
++++ b/CMakeLists.txt	2020-06-24 18:36:15.240828648 +0200
+@@ -161,7 +161,7 @@
+ include_directories("${PROJECT_SOURCE_DIR}/lib/bullet/src")
+ 
+ # Find system ENet library or build it if missing
+-if((UNIX AND NOT APPLE) AND USE_SYSTEM_ENET AND NOT USE_IPV6)
```

Comment:
why removing the check about USE_IPV6 ?

Reply:
libenet doesn't yet support ipv6 upstream https://github.com/lsalzman/enet/pull/109.
So I either disable ipv6 or I use the bundled libenet (which has been fixed to support ipv6).

Reply:
ok but instead of patching, you could probably pass USE_IPV6=no https://notabug.org/Supertuxkart/stk-code/src/master/CMakeLists.txt#L32 ? also maybe we could keep using the bundled enet until the next release with ipv6 support otherwise the update removes feature.

Reply:
In terms of closuze size it's getting worse : 
890.0M before against 898.2M now.

I don't really explain this. It doesn't seem to be static linking since all
libs are dynamically linked.

Reply:
maybe  the system packages have a bigger closure because they are built with more options/stuff ?

Reply:
the difference is little and if one of the lib is reused, it becomes a gain (again :) ) so not an issue for me.

Reply:
Since the bundled libs are statically linked, this frees 2M on the final executable : 
17M→15M